notebooks/centrality/Centrality.ipynb

```diff
@@ -0,0 +1,771 @@
+       "                                <td id=\"T_28f35_row3_col0\" class=\"data row3 col0\" >0.206289</td>\n",
+       "                        <td id=\"T_28f35_row3_col1\" class=\"data row3 col1\" >59.166.0.5</td>\n",
+       "            </tr>\n",
+       "            <tr>\n",
+       "                                <td id=\"T_28f35_row4_col0\" class=\"data row4 col0\" >0.206289</td>\n",
+       "                        <td id=\"T_28f35_row4_col1\" class=\"data row4 col1\" >59.166.0.2</td>\n",
+       "            </tr>\n",
+       "    </tbody></table><style  type=\"text/css\" >\n",
+       "</style><table id=\"T_c21f4_\" style='display:inline'><caption>PageRank</caption><thead>    <tr>        <th class=\"col_heading level0 col0\" >pagerank</th>        <th class=\"col_heading level0 col1\" >vertex</th>    </tr></thead><tbody>\n",
+       "                <tr>\n",
+       "                                <td id=\"T_c21f4_row0_col0\" class=\"data row0 col0\" >0.038591</td>\n",
+       "                        <td id=\"T_c21f4_row0_col1\" class=\"data row0 col1\" >175.45.176.0</td>\n",
+       "            </tr>\n",
+       "            <tr>\n",
+       "                                <td id=\"T_c21f4_row1_col0\" class=\"data row1 col0\" >0.038591</td>\n",
+       "                        <td id=\"T_c21f4_row1_col1\" class=\"data row1 col1\" >175.45.176.2</td>\n",
+       "            </tr>\n",
+       "            <tr>\n",
+       "                                <td id=\"T_c21f4_row2_col0\" class=\"data row2 col0\" >0.038591</td>\n",
+       "                        <td id=\"T_c21f4_row2_col1\" class=\"data row2 col1\" >175.45.176.1</td>\n",
+       "            </tr>\n",
+       "            <tr>\n",
+       "                                <td id=\"T_c21f4_row3_col0\" class=\"data row3 col0\" >0.038591</td>\n",
+       "                        <td id=\"T_c21f4_row3_col1\" class=\"data row3 col1\" >175.45.176.3</td>\n",
+       "            </tr>\n",
+       "            <tr>\n",
+       "                                <td id=\"T_c21f4_row4_col0\" class=\"data row4 col0\" >0.028716</td>\n",
+       "                        <td id=\"T_c21f4_row4_col1\" class=\"data row4 col1\" >10.40.85.1</td>\n",
+       "            </tr>\n",
+       "    </tbody></table>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "print_centrality(5)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There is differences in how each centrality measure ranks the vertices.  In some cases, every algorithms returns similar results and in other cases the results are completely different.  Understand how the centrality measure is computed and understanding what edges represents is key to selecting the right centrality metric.  "
```

Comment:
"There are differences in how each centrality measure ranks the vertices. In some cases, every algorithm returns similar results, and in others, the results are completely different. Understanding how the centrality measure is computed and what edges represent is key to selecting the right centrality metric. "

Reply:
fixed